### DISCUSSION TITLE
SPA Mode

### DISCUSSION THREAD

**Discussion by aimonext** [livewire/livewire#5201] [3. Feature ideas]
Is there any possibility to make transitions without page reloading like SPA without the Turbolink package?

**Comment by siamakabi** [comment]
no yet

**Comment by RealMrHex** [comment]
Technically, this should **not** be added in Livewire.
Do yourself a favor and use InertiaJS which gives you the ability to use frameworks and front-end libraries such as vue and react, and you will also benefit from SSR.

**Reply by joshhanley** [reply]
@RealMrHex well it's coming in V3! 😁

**Reply by RealMrHex** [reply]
> @RealMrHex well it's coming in V3! 😁

It becomes strangely amazing :)
Can u give us some technical info about how it will work in the core? 🤔

**Reply by RealMrHex** [reply]
@joshhanley And when will V3 be available to use?

**Reply by joshhanley** [reply]
@RealMrHex Caleb wrote it, so not 100% on it's implementation, but essentially it will work similar to turbo/turbolinks.

Previously we recommended turbolinks if people wanted the SPA feel, but it wasn't Livewire aware, so it worked but wasn't seamless to use.

So this will be much nicer and be just Livewire using `wire:navigate`.

**Reply by joshhanley** [reply]
@RealMrHex not sure yet, we've still got a fair bit to finish before the beta is ready. But I'm sure as soon as we get close, we'll let everyone know 🙂

**Reply by RealMrHex** [reply]
@joshhanley Can I help in the development of v3? ;)
